src/Akka.CQRS.Hosting.Tests/TradeProcessor.cs

```diff
@@ -2,259 +2,275 @@
+        private readonly string _sqlConnectionString = "Server=sql,1633;User Id=sa;Password=This!IsOpenSource1;TrustServerCertificate=true";
         private readonly string _sqlProviderName = "SqlServer.2019";
+        private string _configTradeProcessor = ConfigurationFactory.ParseString(@"
+            akka {{
+	actor {{
+		provider = cluster
+	}}
+						
+	remote {{
+		dot-netty.tcp {{
+            hostname = ""127.0.0.1""
+            port = 5055
+        }}
+	}}			
 
-        private MsSqlContainer _dockerContainer = null!;
-        private IContainer _lighthouse = null!;
+	cluster {{
+		#will inject this node as a self-seed node at run-time
+		seed-nodes = [""akka.tcp://AkkaTrader@127.0.0.1:5055""] 
+		roles = [""trade-processor"" , ""trade-events""]
 
-        public TradeProcessor(ITestOutputHelper output)
-        {
-            Output = output;
-        }
+		pub-sub{{
+			role = ""trade-events""
+		}}
 
-        public ITestOutputHelper Output { get; }
-        public async Task DisposeAsync()
-        {
-            await _dockerContainer.StopAsync();
-            await _lighthouse.StopAsync();
-        }
+		sharding{{
+			role = ""trade-processor""
+		}}
+	}}
 
-        public async Task InitializeAsync()
-        {
-            #region SQL
-            _dockerContainer = new MsSqlBuilder()
-                .WithName("sql")
-                .WithImage("mcr.microsoft.com/mssql/server:2019-latest")
-                .WithPortBinding(1633, 1433)
-                .WithPassword("This!IsOpenSource1")
-                .WithEnvironment(new Dictionary<string, string>()
-                {
-                        { "ACCEPT_EULA", "Y" }
-                })
-                .WithCleanUp(true)
-                .Build();
+	persistence{{
+		journal {{
+		    plugin = ""akka.persistence.journal.sql""
+		    sql {{
+                event-adapters {{
+                    stock-tagger = ""Akka.CQRS.Infrastructure.StockEventTagger, Akka.CQRS.Infrastructure""
+                }}
+                event-adapter-bindings {{
+                    ""Akka.CQRS.IWithStockId, Akka.CQRS"" = stock-tagger
+                }}
+		    }}
+		}}
 
-            await _dockerContainer.StartAsync();
-            #endregion
-            #region lighthouse
-            _lighthouse = new ContainerBuilder()
-           .WithName("lighthouse")
-           .WithImage("petabridge/lighthouse:latest")
-           .WithPortBinding(9110, 9110)
-           .WithPortBinding(4053, 4053)
-           .WithEnvironment(new Dictionary<string, string>()
-           {
-                { "ACTORSYSTEM", "AkkaTrader" },
-                { "CLUSTER_PORT", "4053"},
-                { "CLUSTER_IP", "lighthouse"},
-                { "CLUSTER_SEEDS", "akka.tcp://AkkaTrader@lighthouse:4053"}
+		snapshot-store {{
+		    plugin = ""akka.persistence.snapshot-store.sql""

```

Comment:
Use Akka.Persistence.SqlServer.Hosting and use the .WithSqlServerPersistence() extension method instead.